post/clang_tidy_review/clang_tidy_review/__init__.py

```diff
@@ -847,21 +855,56 @@ def save_metadata(pr_number: int) -> None:
         json.dump(metadata, metadata_file)
 
 
-def load_review() -> Optional[PRReview]:
-    """Load review output from the standard REVIEW_FILE path.
-    This file contains
+def load_review(review_file) -> Optional[PRReview]:
```

Comment:
Please could you add a type hint for the argument? I know we're not terribly consistent, but it would be nice to add them for new things